Sunrise/ViewModels/CartViewModel.swift

```diff
@@ -119,6 +119,17 @@ class CartViewModel: BaseViewModel {
         return cart.value?.lineItems?[indexPath.row].totalPrice?.description ?? "N/A"
     }
 
+    func productDetailsViewModelForLineItemAtIndexPath(indexPath: NSIndexPath) -> ProductViewModel {
+        var product = ProductProjection()
+        if let lineItem = cart.value?.lineItems?[indexPath.row] {
+            product.id = lineItem.productId
+            // When navigating to PDP from LineItem, we only show one variant
+            product.masterVariant = lineItem.variant
+            product.name = lineItem.name
```

Comment:
Yep, that's very typical thing to do, get everything one size bigger and smaller than your size, of course when there is no returning costs.
PD: I have never bought clothes online, so it's not my own experience :P